mm2src/coins/eth/v2_activation.rs

```diff
@@ -65,6 +70,8 @@ pub enum EthActivationV2Error {
     InvalidHardwareWalletCall,
     #[display(fmt = "Custom token error: {}", _0)]
     CustomTokenError(CustomTokenError),
+    #[from_stringify("WalletConnectError")]
+    WalletConnectError(String),
```

Comment:
Nit: I think `WalletConnectError` should be mapped to other errors, for instance some of the errors will be transport while others due to invalid payload, etc..
P.S. It's fine to keep it this way for now since `EthActivationV2Error` variants don't make much sense and need refactoring.

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2223/commits/3db29387107661f5402f14f0c48d441b20a62e26

Reply:
Typo on refactoring
https://github.com/KomodoPlatform/komodo-defi-framework/blob/ffe7cadd9040c77825429c2c08e0701b7d212064/mm2src/coins/eth/v2_activation.rs#L73